### ISSUE TITLE
Lagrangian Mechanics

### ISSUE THREAD

**Issue by bennn** [bennn/mechanics#13]
Work through [chapter 1](http://mitpress.mit.edu/sites/default/files/titles/content/sicm/book-Z-H-7.html#%_chap_1) of the book. Add files from scmutils as needed to get the examples & exercises working.

(See branch `lagrangian-mechanics`)

**Comment by bkc39** [comment]
I think the real difficulty here is that we need to implement the `display` library and in particular `show-expression` and `print-expression`. This looks like a non-trivial task and should probably be its own separate issue.

As far as I can tell we need to implement three components of this
- `structures`: where they define `up` and `down` among many other things
- `print`: which is the translation from S-expressions to/from TeX
- `display`: where we actually render the TeX in the Racket shell

I read their code last night. I think that all three of these pieces are in need of some serious internal "modernization". We should probably set this as a milestone and break it into sub-issues that we can  work on separately.

**Comment by bennn** [comment]
Yeah that's why I started #12. I want to ignore this problem for now. If there's no useful Racket library to help with this, it should probably be an independent repo.

**Comment by bkc39** [comment]
I think this issue should be re-purposed to basically be a port of the example code in Ch1 to verify that our library is fully backward compatible.
